src/facetgroups/CheckboxFacetGroup.js

```diff
@@ -30,12 +34,17 @@ class CheckboxFacetGroup extends React.Component {
   }
 
   render() {
-    const {facetGroupName, facets} = this.props
+    const {facetGroupName, facetGroupNameDescription, facets} = this.props
     const {checkedFacets} = this.state
 
     return (
       <div className={`padding-bottom-xlarge`}>
-        <h4>{facetGroupName}</h4>
+        <h4>{facetGroupName}
+        {facetGroupNameDescription != null &&
+          <span>
+    <sup data-tooltip aria-haspopup="true" className="has-tip tip-right" style={tooltipStyle} title={facetGroupNameDescription}>?</sup>
+    </span>
+  }</h4>
```

Comment:
Clean this up. The Enter and Tab keys are your friends. 😜 Also, be consistent and stick to the style `className={class-in-backquotes}` instead of `className="class"`.